updating/preparing_for_updates/updating-cluster-prepare.adoc

```diff
@@ -12,7 +12,12 @@ WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to
 To do: Remove this comment once 4.13 docs are EOL.
 ////
 
-Learn more about administrative tasks that cluster admins must perform to successfully initialize an update, as well as optional guidelines for ensuring a successful update.
+Cluster admins must complete certain administrative tasks to successfully initialize an update. Consider using optional guidelines for ensuring a successful update.
+
+[NOTE]
+====
+For a cluster that runs on {vmw-first}, you can use vSphere automatic migration to provision in-tree storage plugins to their counterpart Container Storage Interface (CSI) drivers. For more information, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-sc-vsphere_persistent-storage-csi-migration[vSphere automatic migration].
```

Comment:
```suggestion
For a cluster that runs on {vmw-first}, you can use vSphere CSI automatic migration to provision in-tree storage plugins to their counterpart Container Storage Interface (CSI) drivers. For more information, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-sc-vsphere_persistent-storage-csi-migration[vSphere automatic migration].
```

Hi, let's use "vSphere CSI automatic migration" instead of "vSphere automatic migration" as far as possible. (which means only vSphere PV volume migration)